### PR TITLE
Add matplotlib scalebar

### PR DESCRIPTION
**Pull request by ericpre**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/matplotlib-scalebar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "Artist for matplotlib to display a scale bar"
+  doc_url: "https://github.com/ppinard/matplotlib-scalebar"
+  dev_url: "https://github.com/ppinard/matplotlib-scalebar"
+
+extra:
+  recipe-maintainers:
+    - ppinard
```
Comment (ericpre) [inline recipes/matplotlib-scalebar/meta.yaml]: @ppinard: are you happy with being a maintainer of this recipe?

**Comment by ericpre** [comment]
CI fails because of missing license: https://github.com/ppinard/matplotlib-scalebar/pull/24 should fix it.

File: recipes/matplotlib-scalebar/meta.yaml (hunk 1 above)
Comment (ppinard) [reply recipes/matplotlib-scalebar/meta.yaml]: Sure. Thank you for creating the recipe.

**Comment by ppinard** [comment]
I just released version 0.6.0 with the LICENSE file in the tarball.

**Comment by ericpre** [comment]
@conda-forge/staged-recipes: this is ready for review!

**Pull request merged**